src/cascadia/TerminalSettingsEditor/AddProfile.xaml

```diff
@@ -23,27 +23,31 @@
 
     <ScrollViewer ViewChanging="ViewChanging">
         <StackPanel Style="{StaticResource SettingsStackStyle}">
-            <Button x:Uid="AddProfile_AddNewButton"
-                    AutomationProperties.AutomationId="AddProfile_AddNewButton"
-                    AutomationProperties.Name="{Binding Tag, RelativeSource={RelativeSource Self}}"
-                    Click="AddNewClick"
-                    Style="{StaticResource AccentButtonStyle}">
-                <Button.Content>
-                    <StackPanel Orientation="Horizontal">
-                        <FontIcon FontSize="{StaticResource StandardIconSize}"
-                                  Glyph="&#xE710;" />
-                        <TextBlock x:Uid="AddProfile_AddNewTextBlock"
-                                   Style="{StaticResource IconButtonTextBlockStyle}" />
-                    </StackPanel>
-                </Button.Content>
-            </Button>
+            <Border MaxWidth="{StaticResource StandardControlMaxWidth}">

```

Comment:
can these two areas not be in the same Border?

Reply:
A border can only have 1 child, so the only way would be to put the stack panel inside the border but that causes this to happen
<img width="899" alt="stackpaneloff" src="https://user-images.githubusercontent.com/26824113/152217721-3c25f8d3-6f96-448c-b34e-521eb6c404d8.png">

Reply:
oh right.

i'm surprised about that result, but... xaml